app.py

```diff
@@ -1,32 +1,195 @@
-from flask import Flask
-
+import csv
 import pymongo
+from flask import Flask, render_template_string, request
+import plotly.graph_objects as go
+import plotly.io as pio
 import pandas as pd
-from pymongo import MongoClient
+import plotly.express as px
+"""
 
+"""
 app = Flask(__name__)
 
+# Initialize a connection to the MongoDB database
+mongo_client = pymongo.MongoClient("mongodb://root:pass@localhost:27018")
+db = mongo_client["my_database"]
+collection = db["spotify_data"]
```

Comment:
need to define a global thing for spotify data. 
so we don't have to change with each step